app/views/games/show.html.erb

```diff
@@ -28,95 +63,96 @@
-        <div class="white left"><%= render_piece(6,2) %></div>
-        <div class="black left"><%= render_piece(7,2) %></div>
-      </div>
-      <div class="row">
-        <div class="black left"><%= render_piece(0,1) %></div>
-        <div class="white left"><%= render_piece(1,1) %></div>
-        <div class="black left"><%= render_piece(2,1) %></div>
-        <div class="white left"><%= render_piece(3,1) %></div>
-        <div class="black left"><%= render_piece(4,1) %></div>
-        <div class="white left"><%= render_piece(5,1) %></div>
-        <div class="black left"><%= render_piece(6,1) %></div>
-        <div class="white left"><%= render_piece(7,1) %></div>
-      </div>
+        <div class="row">
+          <div class="white left"><%= render_piece(0,2) %></div>
+          <div class="black left"><%= render_piece(1,2) %></div>
+          <div class="white left"><%= render_piece(2,2) %></div>
+          <div class="black left"><%= render_piece(3,2) %></div>
+          <div class="white left"><%= render_piece(4,2) %></div>
+          <div class="black left"><%= render_piece(5,2) %></div>
+          <div class="white left"><%= render_piece(6,2) %></div>
+          <div class="black left"><%= render_piece(7,2) %></div>
+        </div>
+        <div class="row">
+          <div class="black left"><%= render_piece(0,1) %></div>
+          <div class="white left"><%= render_piece(1,1) %></div>
+          <div class="black left"><%= render_piece(2,1) %></div>
+          <div class="white left"><%= render_piece(3,1) %></div>
+          <div class="black left"><%= render_piece(4,1) %></div>
+          <div class="white left"><%= render_piece(5,1) %></div>
+          <div class="black left"><%= render_piece(6,1) %></div>
+          <div class="white left"><%= render_piece(7,1) %></div>
+        </div>
 
-      <div class="row bottom">
-        <div class="white left"><%= render_piece(0,0) %></div>
-        <div class="black left"><%= render_piece(1,0) %></div>
-        <div class="white left"><%= render_piece(2,0) %></div>
-        <div class="black left"><%= render_piece(3,0) %></div>
-        <div class="white left"><%= render_piece(4,0) %></div>
-        <div class="black left"><%= render_piece(5,0) %></div>
-        <div class="white left"><%= render_piece(6,0) %></div>
-        <div class="black left"><%= render_piece(7,0) %></div>
+        <div class="row bottom">
+          <div class="white left"><%= render_piece(0,0) %></div>
+          <div class="black left"><%= render_piece(1,0) %></div>
+          <div class="white left"><%= render_piece(2,0) %></div>
+          <div class="black left"><%= render_piece(3,0) %></div>
+          <div class="white left"><%= render_piece(4,0) %></div>
+          <div class="black left"><%= render_piece(5,0) %></div>
+          <div class="white left"><%= render_piece(6,0) %></div>
+          <div class="black left"><%= render_piece(7,0) %></div>
```

Comment:
Inside render piece, you'll want to add something like...

`link_to ..., path..., data: { update_url: piece_path(piece) }`

Can pass data attributes to a link_to. Then when you "drag" the link. Jquery will be able to access the attributes.

`piece_path(piece)` will return in `/pieces/5` (for example). Then when you do a PUT to it. You'll hit `pieces_controller#update`